docs/source/bfiq.rst

```diff
@@ -231,8 +231,7 @@ The file fields in the bfiq array files are:
 | | max_num_sequences]              | | These timestamps come back from the USRP  | 
 | |                                 | | driver and the USRPs are GPS disciplined  |
 | |                                 | | and synchronized using the Octoclock.     |
-| |                                 | | Provided in milliseconds since epoch.     | 
-| |                                 | | Note that records that do not have        | 
+| |                                 | | Provided in  that do not have             |
```

Comment:
```suggestion
| |                                 | | Provided in seconds since epoch.          |
| |                                 | | Note that records do not have             |
```